tests/integration/test_dummy.py

```diff
@@ -2,7 +2,7 @@
 
 import pytest
 
-from adcm_aio_client.core.client import ADCMClient
+from adcm_aio_client.core.client._client import ADCMClient
```

Comment:
why introducing _client?

Reply:
To make clear, that `build_client` is the only convenient way to get an ADCMClient

Reply:
returned core/client.py